cross/libgpg-error/Makefile

```diff
@@ -11,6 +11,13 @@ HOMEPAGE = http://www.gnupg.org/related_software/libgpg-error/
 COMMENT  = Libgpg-error is a small library that defines common error values for all GnuPG components
 LICENSE  = LGPL
 
+PRE_CONFIGURE_TARGET = preConfigureTarget
 GNU_CONFIGURE = 1
 
 include ../../mk/spksrc.cross-cc.mk
+
+.PHONY: preConfigureTarget
+preConfigureTarget:
+	cp $(WORK_DIR)/$(PKG_DIR)/src/syscfg/lock-obj-pub.arm-unknown-linux-gnueabi.h $(WORK_DIR)/$(PKG_DIR)/src/syscfg/lock-obj-pub.linux-gnueabi.h
```

Comment:
This probably won't work for all arches. But I really don't know how to improve that.
I had an error when cross compiling gnupg : ``./gpg-error.h.in:320: error including `./syscfg/lock-obj-pub.linux-gnueabi.h': No such file or directory``
I found this partial solution on https://github.com/OpenELEC/OpenELEC.tv/blob/9df5e4f92a113c7d2fdc2cff8a1757f70103299f/packages/security/libgpg-error/package.mk